pandas/tests/io/test_parquet.py

```diff
@@ -448,6 +448,11 @@ class TestParquetFastParquet(Base):
     def test_basic(self, fp, df_full):
         df = df_full
 
+        # additional supported types for fastparquet >= 0.1.4
```

Comment:
pls follow the existing style. see how we have a fixture for pa_ge_070. you need to make similar for fp

Reply:
good point!

Reply:
little confused. But I agree it's better to match existing style in test_basic to avoid unnecessary style variation.

Reply:
this is a totally incorrect style ATM. So please see what you can fix. skipping INSIDE a test function is not correct, rather a fixture should be used. This must have slipped in.

Reply:
Jeff, please leave this. I asked @minggli to do this, and it is the current style. You can do a follow-up PR if you want to change this in all places.